pkg/kor/all.go

```diff
@@ -23,6 +23,11 @@ type ResourceDiff struct {
 	diff         []string
 }
 
+type ResourceDiff2 struct {
```

Comment:
why a new type?

Reply:
This is temporary until I switch them all